include/xrpl/protocol/detail/transactions.macro

```diff
@@ -447,6 +447,13 @@ TRANSACTION(ttCREDENTIAL_DELETE, 60, CredentialDelete, ({
     {sfCredentialType, soeREQUIRED},
 }))
 
+/** This transaction type modify a NFToken */
+TRANSACTION(ttNFTokenModify, 61, NFTokenModify, ({
```

Comment:
🤔  This name seemed changed. Why `ttNFTokenModify` is using different naming style?

Reply:
Good catch! fixed!